src/common/components/StatusCard.jsx

```diff
@@ -269,6 +274,8 @@ const StatusCard = ({ deviceId, position, onClose, disableActions, desktopPaddin
           <MenuItem component="a" target="_blank" href={`https://www.google.com/maps/search/?api=1&query=${position.latitude}%2C${position.longitude}`}>{t('linkGoogleMaps')}</MenuItem>
           <MenuItem component="a" target="_blank" href={`http://maps.apple.com/?ll=${position.latitude},${position.longitude}`}>{t('linkAppleMaps')}</MenuItem>
           <MenuItem component="a" target="_blank" href={`https://www.google.com/maps/@?api=1&map_action=pano&viewpoint=${position.latitude}%2C${position.longitude}&heading=${position.course}`}>{t('linkStreetView')}</MenuItem>
+          {serverNavigationAppLink && serverNavigationAppTitle && <MenuItem component="a" target="_blank" href={serverNavigationAppLink.replace('{latitude}', position.latitude).replace('{longitude}', position.longitude)}>{serverNavigationAppTitle}</MenuItem>}
+          {userNavigationAppLink && userNavigationAppTitle && <MenuItem component="a" target="_blank" href={userNavigationAppLink.replace('{latitude}', position.latitude).replace('{longitude}', position.longitude)}>{userNavigationAppTitle}</MenuItem>}
```

Comment:
I would just check one of these attributes instead of both. Maybe title.

Reply:
Fixed ✅